### PR TITLE
2385 Badly formatted character/charset support code

### PR DESCRIPTION
**Pull request by astares**
Fix for https://github.com/pharo-project/pharo/issues/2385

### CODE REVIEW

File: src/Kernel-Tests-Extended/AsciiCharsetTest.class.st [hunk 1]
```diff
@@ -28,15 +28,16 @@ AsciiCharsetTest >> testIsCasedLetter [
...
-
 	"Exhaustively check that the charset under test behaves like Unicode"
 
 	| charset |
 	charset := self classUnderTest.
 	0 to: charset maxValue do: [ :asciiValue | 
 		| ch |
 		ch := Character value: asciiValue.
-		self assert: (Unicode isClosePunctuation: ch) equals: (charset isClosePunctuation: ch) ]
+		self
```
Comment (jecisc) [inline src/Kernel-Tests-Extended/AsciiCharsetTest.class.st:38]: I am not a big fan of all the line returns in those tests. Almost everyone working on Pharo have at least a 13', and those lines are for frow wrapping on a 13'.

**Comment by pavel-krivanek** [comment]
pharoscreenshot

I do not know what is wrong with such asserts that they are necessary to reformat... It's standard window size.

File: src/Kernel-Tests-Extended/AsciiCharsetTest.class.st (hunk 1 above)
Comment (astares) [reply src/Kernel-Tests-Extended/AsciiCharsetTest.class.st:38]: Then blame/repair the standard formatter.

**Comment by astares** [comment]
Stef is working on a new pretty printer, at least as time allows

code in 
	https://github.com/Ducasse/mybecher/tree/master/BlueInk-SuperTests/

**Comment by Ducasse** [comment]
Failed tests are unrelated.

**Pull request merged**